04week/mastermind.js

```diff
@@ -28,13 +29,41 @@ function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
-  // your code here
+function generateHint(solution, guess) {
```

Comment:
this should be `function generateHint(guess) {` instead, I know the textbook said to, but you wont be able to pass the tests if you do that (almost everyone had this problem)